utils/requests.ts

```diff
@@ -60,6 +60,10 @@ export async function getUpdatesRequest({req}) {
     
     if (req.query.filter && !["all", "drafts", "published"].includes(req.query.filter)) conditions["tags"] = req.query.filter;
 
+    // Technically, this is unrobust because it checks for the exact time being 00:00:00.000Z, but all updates have this so :shrug: it works
```

Comment:
oh jeez i didnt realize it was even a full time string in the date field. it comes from the manual date selector so its always only a date

this is probably fine for now

Reply:
ya i think it's bc every js `Date` object has a time attached to it so since none is specified, it just sets 0 utc